src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/MSBuildRestoreUtility.cs

```diff
@@ -951,5 +939,76 @@ public static Task ReplayWarningsAndErrorsAsync(IEnumerable<IAssetsLogMessage> m
 
             return logger.LogMessagesAsync(logMessages);
         }
+
+        private static Dictionary<NuGetFramework, Dictionary<string, CentralVersionDependency>> CreateCentralVersionDependencies(IEnumerable<IMSBuildItem> items,
+            IList<TargetFrameworkInformation> specFrameworks)
+        {
+            var centralVersionDependencies = GetItemByType(items, "CentralVersionDependency")?.Distinct(new MSBuildItemComparer()).ToList();
+            var result = new Dictionary<NuGetFramework, Dictionary<string, CentralVersionDependency>>();
+
+            foreach (var cvd in centralVersionDependencies)
```

Comment:
Use the type name, see : [Rule #10](https://github.com/NuGet/NuGet.Client/blob/715edd1291b089d42cfef65566146f45a8cd6310/docs/coding-guidelines.md)

Reply:
Updated.